projects/packages/forms/src/dashboard/landing/style.scss

```diff
@@ -0,0 +1,344 @@
+.jp-forms__landing {
```

Comment:
Will we also need WP.com styles for this?

Reply:
@digitalwaveride, will we need it?

Reply:
We will work with a different screenshot for wp.com ... but keep the JP branding.